src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
```

Comment:
2 [NAMING] - use proper names for variables - if you want to name variable result, pick a better one :)

Reply:
Thanks for the hint, but the `result` are written in the tests, so it can't be another var